pkg/scheduler/objects/queue.go

```diff
@@ -113,9 +113,29 @@ func newBlankQueue() *Queue {
 	}
 }
 
-// NewConfiguredQueue creates a new queue from scratch based on the configuration
+// NewConfiguredQueue creates a new queue from scratch based on the configuration and logs at info level
 // lock free as it cannot be referenced yet
 func NewConfiguredQueue(conf configs.QueueConfig, parent *Queue) (*Queue, error) {
+	queue, err := newConfiguredQueueInternal(conf, parent)
+	if queue != nil {
+		log.Log(log.SchedQueue).Info("configured queue added to scheduler",
+			zap.String("queueName", queue.QueuePath))
+	}
+	return queue, err
+}
+
+// NewConfiguredShadowQueue creates a new queue from scratch based on the configuration and logs at debug level
```

Comment:
Hi @kousei47747, thank you for your contribution!

Could you provide more details about the differences between `NewConfiguredQueue` and `NewConfiguredShadowQueue`?
Currently, it appears the only difference is the log level, which may be confusing for those unfamiliar with shadow queues.

It would be helpful to explain when developers should use `NewConfiguredQueue` versus `NewConfiguredShadowQueue`.

Reply:
@blueBlue0102 thanks for reviewing! I will provide more detailed description in the new patch.